test/export/test_export.py

```diff
@@ -9500,7 +9500,7 @@ def forward(self, x):
                 return x[:out_size]
 
         dim_min = 5
-        dim_max = 10
+        dim_max = 8
```

Comment:
I had to do this because changing the slice comparison resulted in the following single-symbol guard: `round(((float(L['x'].size()[0])) / 2.0)) < L['x'].size()[0]`, which is obviously true. However, we can't get to that conclusion with only value range analysis, because we end up with: `[2, 5] < [5, 10]`.

Reply:
Reverted. Not needed after replacing the guards tweak with the static checks.